integrations/Braze/browser.js

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable consistent-return */
 /* eslint-disable class-methods-use-this */
+import cloneDeep from "lodash.clonedeep";
+import isEqual from "lodash.isequal";
 import { del } from "obj-case";
 import logger from "../../utils/logUtil";
```

Comment:
Can you update the logger here with the new logger? You can refer [here](https://github.com/rudderlabs/rudder-sdk-js/pull/657). And also update the canonical name functionality similar to [this](https://github.com/rudderlabs/rudder-sdk-js/pull/685).

Reply:
yeah, updated the logger.
And for the Canonical name, the PR is reverted, currently, this is not in v1-staging.